src/styles/ForecastSummary.css

```diff
@@ -0,0 +1,30 @@
+.forecast-summary{
+  padding: 5%;
+  background: #66748A;
+  border-radius: 25px;
+  
+  color:white;
+}
+
+.forecast-summary__button{
+  box-shadow: 0px 0px 0px transparent;
+  border: 0px solid transparent;
+  background:transparent;
+  text-shadow: 0px 0px 0px transparent;
```

Comment:
Why not add `color:#66748A;` here? The button class in this file can then be removed